ThunderstoreCLI/Options.cs

```diff
@@ -6,7 +6,37 @@
 namespace ThunderstoreCLI.Options;
 
 /// Options are arguments passed from command line.
-public abstract class PackageOptions
+public abstract class BaseOptions
+{
+    [Option("output", Required = false, HelpText = "The output format for all output. Valid options are HUMAN and JSON.")]
+    public InteractionOutputType OutputType { get; set; } = InteractionOutputType.HUMAN;
+
+    [Option("tcli-directory", Required = false, HelpText = "Directory where TCLI keeps its data, %APPDATA%/ThunderstoreCLI on Windows and ~/.config/ThunderstoreCLI on Linux")]
+    // will be initialized in Init if null
+    public string TcliDirectory { get; set; } = null!;
+
+    public virtual void Init()
+    {
+        InteractionOptions.OutputType = OutputType;
+
+        // ReSharper disable once ConstantNullCoalescingCondition
+        TcliDirectory ??= Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "ThunderstoreCLI");
+    }
+
+    public virtual bool Validate()
```

Comment:
Not sure this should be called `Validate` as it doesn't actually validate anything?

Reply:
Not sure what a better name might be, and the derived implementations do validate some things.